.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on: [pull_request, push]
```

Comment:
Would it be better to limit the target branches as explained in: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration ?

Reply:
Ah right, `pull_request` should be limited to `master`

Reply:
Ah no, specifying `master` for both should be fine?